tests/ci/cdk/cdk/aws_lc_github_ci_stack.py

```diff
@@ -58,14 +58,4 @@ def __init__(self,
                                                    privileged=False,
                                                    build_image=codebuild.LinuxBuildImage.STANDARD_4_0),
             build_spec=codebuild.BuildSpec.from_object(build_spec_content))
-
-        # TODO: add build type BUILD_BATCH when CFN finishes the feature release. See CryptoAlg-575.
-
-        # Add 'BuildBatchConfig' property, which is not supported in CDK.
-        # CDK raw overrides: https://docs.aws.amazon.com/cdk/latest/guide/cfn_layer.html#cfn_layer_raw
-        # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-codebuild-project.html#aws-resource-codebuild-project-properties
-        cfn_build = project.node.default_child
-        cfn_build.add_override("Properties.BuildBatchConfig", {
-            "ServiceRole": role.role_arn,
-            "TimeoutInMins": 180
```

Comment:
Nit: is this carried on? Or the default setting can work for us?

Reply:
I did some poking around and after a new propagation to my fork, I noticed that Line 56 defines the timeout value for the codebuild project. The Codebuild batch inherits this value when using `project.enable_batch_builds()`.
Same concept goes for the other files below.